packages/integration/src/azure/AzureIntegration.ts

```diff
@@ -42,4 +42,32 @@ export class AzureIntegration implements ScmIntegration {
   get config(): AzureIntegrationConfig {
     return this.integrationConfig;
   }
+
+  /*
+   * Azure repo URLs on the form with a `path` query param are treated specially.
+   *
+   * Example base URL: https://dev.azure.com/organization/project/_git/repository?path=%2Fcatalog-info.yaml
+   */
+  resolveUrl(url: string, base: string): string {
```

Comment:
Wondering if it makes more sense to make this `resolveUrl(base: string, relative: string)`? Feeling a bit more intuitive since that's how `path.resolve` works. Also would allow us to make the relative bit a vararg.

Reply:
On the other hand this is exactly the URL constructor names and order :) which is the one I think makes more sense to mimic. This one is also meant to be specialised for that two-arg use case, so probably no reason for a vararg

Reply:
Hey you know what. This is exactly the kind of method that should take a single object arg, way clearer. `{ url, base }`